components/browser/state/src/main/java/mozilla/components/browser/state/reducer/ShareInternetResourceStateReducer.kt

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.state.reducer
+
+import mozilla.components.browser.state.action.ShareInternetResourceAction
+import mozilla.components.browser.state.state.BrowserState
+import mozilla.components.browser.state.state.ContentState
+
+internal object ShareInternetResourceStateReducer {
```

Comment:
Could we add some tests to covered this class? See https://github.com/mozilla-mobile/android-components/pull/9420/checks?check_run_id=1747490825

Reply:
Will do.